Directory.Build.Props

```diff
@@ -7,7 +7,7 @@
     <Authors>VirtoCommerce</Authors>
   </PropertyGroup>
    <PropertyGroup>
-    <VersionPrefix>3.2.0</VersionPrefix>
+    <VersionPrefix>3.2.1</VersionPrefix>
```

Comment:
You mustn't increment platform version by your own.

Reply:
See PR description: 
1. Update platform version and merge to master as part of hotfix strategy

Ask @lnetrebskii for more information